docker-compose.yml

```diff
@@ -21,6 +21,11 @@ console:
     - logplex.env
   volumes_from:
     - base
+  volumes:
+    - ./_build:/usr/src/app/_build
```

Comment:
See my comments below about the local _build directory. However, I would also add the local config directory to this list.